src/GafferSceneUI/SceneGadget.cpp

```diff
@@ -657,8 +657,8 @@ std::optional<V3f> SceneGadget::normalAt( const IECore::LineSegment3f &lineInGad
 	// incorrect. We know that the incoming `lineInGadgetSpace` has been calculated from
 	// an integeer pixel value so we snap to the nearest integer as a corrective.
```

Comment:
integeer -> integer. Although it might make more sense to talk about pixel centres than integers now :

> We expect that `lineInGadgetSpace` corresponds to the centre of a pixel, so we snap to the nearest center as a corrective.